sample/src/main/java/com/thoughtworks/ark/sample/permission/PermissionSampleActivity.kt

```diff
@@ -0,0 +1,17 @@
+package com.thoughtworks.ark.sample.permission
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import com.thoughtworks.ark.ui.theme.Theme
+
+class PermissionSampleActivity : AppCompatActivity() {
```

Comment:
Move this `Activity` and `SampleUI` to `ui` package, we will ignore the test coverage theree

Reply:
It's better to rename it to `PermissionActivity`